jddd-architecture/jddd-architecture-onion/src/main/java/org/jddd/architecture/onion/DomainModelRing.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jddd.architecture.onion;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Identifies the {@link DomainModelRing} in an onion architecture. The domain model ring is the inner-most ring in the
+ * onion architecture and is only coupled to itself. It models the truth of the business domain by consisting of state
```

Comment:
I usually focus more on the behaviour and would suggest writing "consisting of behaviour (logic) and the required state (data)"

Reply:
agree